### DISCUSSION TITLE
How to solve: `Connectivity check failed due to DNS Lookup Failed`

### DISCUSSION THREAD

**Discussion by gbraad** [spotsnel/cloudflare-client#2] [Show and tell]
You might get the following issue on Fedora:

```
$ warp-cli connect
Success
$ warp-cli status
Status update: Connecting
Success
$ warp-cli status
Success
Status update: Unable to connect. Reason: Connectivity check failed due to DNS Lookup Failed
```

Add the following line to /etc/systemd/resolved.conf:
```ini
ResolveUnicastSingleLabel=yes
```

Restart the service:
```shell
sudo systemctl restart systemd-resolved.service
```

```
$ warp-cli status
Success
Status update: Connected
```

Ref: https://developers.cloudflare.com/cloudflare-one/faq/teams-troubleshooting/#warp-on-linux-shows-dns-connectivity-check-failed-with-reason-dnslookupfailed

**Comment by bestperson-free** [comment]
Hi, this did not help, it did not help https://bbs.archlinux.org/viewtopic.php?id=301112, and this is https://bbs.archlinux.org/viewtopic.php?id=287445. I had closer to 2 links, although it is natural and what is higher. So I understood (probably), a encrypted DNS TLS, HTTPS has been installed on the router, the Systemd-ReSolved.Service is the same (although the shutdown in it did not help). We do it 

Or
```
warp-cli mode warp+doh 
```
Or
to comment out  /etc/systemd/resolved.conf  

```
#DNSOverTLS=yes
```
The last option is preferable

And here it is ResolveUnicastSingleLabel=yes I have it disabled
And everything is allowed)
Manjaro KDE
